### PR TITLE
Changelog entry for v0.1.2

### PR DESCRIPTION
**Pull request by weiji14**
**Description of proposed changes**

Patch release in preparation for the Scipy 2020 sprint session. Preview it live at https://pygmt-git-changelog-v012.gmt.now.sh/changes.html. For reference, the previous changelog is at #445.

Changelog made by following the instructions at https://github.com/GenericMappingTools/pygmt/blob/master/MAINTENANCE.md#updating-the-changelog. The changelog itself includes the following sections:

* Highlights
* New features
* Enhancements
* Documentation
* Bug fixes
* Maintenance
* List of Contributors (sorted by number of commits)

TODO:
- [x] Format Pull Request links using ``sed -i.bak -E 's$\(#([0-9]*)\)$(`#\1 <https://github.com/GenericMappingTools/pygmt/pull/\1>`__)$g' changes.rst``.
- [x] Add Zenodo DOI link once it's reserved.
- [x] Add in new Pull Requests merged in for [v0.1.2](https://github.com/GenericMappingTools/pygmt/milestones/0.1.x)

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

### CODE REVIEW

File: doc/changes.rst [hunk 1]
```diff
@@ -1,6 +1,58 @@
 Changelog
 =========
 
+Release v0.1.2 (2020/07/XX)
```
Comment (seisman) [inline doc/changes.rst:4]: Remember to update the date before merging.

File: doc/changes.rst [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+   :target: https://doi.org/10.5281/zenodo.3930577
+
+Highlights:
+
+* Patch release in preparation for the Scipy 2020 sprint session
+* Last version to support GMT 6.0, future PyGMT versions will require GMT 6.1 or newer
+
+New Features:
+
+* Wrap grdcut (`#492 <https://github.com/GenericMappingTools/pygmt/pull/492>`__) @seisman
```
[the remark this answers is not in the document]
Comment (weiji14) [reply doc/changes.rst:18]: Yeah, I'll remove it from the Release Drafter yml specification too.

**Comment by weiji14** [comment]
One last thing. We should have done this in v0.1.1, but I feel like we should add @moorepants to the Contributor list for his help on setting up the conda-forge pygmt package.

**Comment by seisman** [comment]
Yes, of course.

**Pull request merged**